### ISSUE TITLE
Why is zeronet only using the first 64 chars of the sha 512 hash?

### ISSUE THREAD

**Issue by mkg20001** [HelloZeroNet/ZeroNet#1074]
Take for example zerohello
```
  "template-new/js/ZeroFrame.js": {
   "sha512": "8bc18006f7bb457cca2338239ea345bc1db54a664b4bc9b8a34f4e4133d1776b",
   "size": 3816
  }
```
the hash only has a length of 64

while `sha512sum template-new/js/ZeroFrame.js` outputs
```
8bc18006f7bb457cca2338239ea345bc1db54a664b4bc9b8a34f4e4133d1776b90ba13b8117e24a325c18c7a36e82f2c53208b7f639508f1edf8c598d8b22b6d  template-new/js/ZeroFrame.js
```

**Comment by HelloZeroNet** [comment]
Because it's safe enough, requires less storage and on 64bit cpus faster, than sha256. (see: https://crypto.stackexchange.com/questions/3153/sha-256-vs-any-256-bits-of-sha-512-which-is-more-secure)

**Comment by JasonTsou** [comment]
mute发自我的iPhone------------------ Original ------------------From: ZeroNet  <notifications@github.com>Date: Sat,Aug 12,2017 17:51To: HelloZeroNet/ZeroNet <ZeroNet@noreply.github.com>Cc: Subscribed <subscribed@noreply.github.com>Subject: Re: [HelloZeroNet/ZeroNet] Why is zeronet only using the first 64chars of the sha 512 hash? (#1074)Closed #1074.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/HelloZeroNet/ZeroNet","title":"HelloZeroNet/ZeroNet","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/HelloZeroNet/ZeroNet"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Closed #1074."}],"action":{"name":"View Issue","url":"https://github.com/HelloZeroNet/ZeroNet/issues/1074#event-1203567641"}}}
